constraint_utils/constraint_basics.py

```diff
@@ -339,6 +370,19 @@ def calculate_mu_by_downsampling(genome_ht: hl.Table, raw_context_ht: hl.MatrixT
 def get_proportion_observed_by_coverage(exome_ht: hl.Table, context_ht: hl.Table, mutation_ht: hl.Table,
                                         recompute_possible: bool = False, dataset: str = 'gnomad',
                                         impose_high_af_cutoff_upfront: bool = True) -> hl.Table:
+    """Compute the proportion obsered by exome coverage 
```

Comment:
is this actually a proportion or just the count observed?

Reply:
It counts the possible variants and observed variants, but I don't see it calculate proportion.